### ISSUE TITLE
About Dataloader

### ISSUE THREAD

**Issue by tonysy** [CSAILVision/semantic-segmentation-pytorch#44]
Hi @Tete-Xiao @hangzhaomit ,thanks for your  work.
I have a question about dataloader.

Why you do `batch_segms = batch_segms-1` to change label from `0->150` to `-1->149` and why use BGR instead of RGB?.

 If I use another dataset, Do I need to change the segms alse and use BGR mode? Thansk a lot.

**Comment by Tete-Xiao** [comment]
The answer to the first two questions is "Because of Roman horse’s ass" lol.
 You can refer to [https://www.learnopencv.com/why-does-opencv-use-bgr-color-format/](url) for the details.

And the third question, apparently, yes.
